compiler/circle-interpreter/src/CircleInterpreter.cpp

```diff
@@ -104,8 +104,7 @@ int entry(int argc, char **argv)
   const auto output_prefix = arser.get<std::string>("output_prefix");
 
   // Load model from the file
-  luci::ImporterEx importer;
-  std::unique_ptr<luci::Module> module = importer.importVerifyModule(filename);
+  std::unique_ptr<luci::Module> module = luci::importVerifyModule(filename);
```

Comment:
This line demonstrates how the new utility can be used in ONE to import models using the `ImporterEx` underneath.

Going with the new free function minimizes the impact on the code in ONE (most of the cases it's a one-liner) and does not require adding a new method to the public API of `ImporterEx`.